spec/factories/course_forum_topics.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :forum_topic, class: Course::Forum::Topic.name do
-    forum
+    transient do
+      course { build(:course) }
```

Comment:
There is no association named `course_discussion_topic ` in `forum_topic`, that does not work ? 

I would say:
1. Remove the discussion topic factory and keep the `course` as a transient for `forum_topic`
2. Or leave the discussion topic there, and make it parent of the the `forum_topic`

what do you think ?